deploy/FullDeployTest.js

```diff
@@ -268,6 +268,7 @@ module.exports = async function (hre) {
         "deployTracer",
         deployTracerData,
         oracleAdapter.address,
+        gasPriceOracle.address,
```

Comment:
```suggestion
        gasOracle.address,
```

Since `gasPriceOracle` is the quoteToken/gas oracle.